lib/Service/ParticipantService.php

```diff
@@ -426,9 +433,25 @@ public function addUsers(Room $room, array $participants, ?IUser $addedBy = null
 			$this->dispatcher->dispatchTyped($attendeeEvent);
 
 			$this->dispatcher->dispatch(Room::EVENT_AFTER_USERS_ADD, $event);
+
+			$lastMessage = $event->getLastMessage();
+			if ($lastMessage instanceof IComment) {
+				$this->updateRoomLastMessage($room, $lastMessage);
+			}
 		}
 	}
 
+	protected function updateRoomLastMessage(Room $room, IComment $message): void {
```

Comment:
Is this method only for update cache? If true, maybe will be good expose this at name of method
```suggestion
	protected function updateCacheOfRoomLastMessage(Room $room, IComment $message): void {
```

Reply:
It's not only about the cache but also about the database